sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentProducer.java

```diff
@@ -207,7 +212,7 @@ public DocumentProducer(
                         if (logger.isDebugEnabled()) {
                             logger.info("Cross Partition Query Execution detected partition [{}] split into [{}] partitions,"
                                     + " last continuation token is [{}].",
-                                    targetRange.toJson(),
+                                    feedRange.toJson(),
```

Comment:
for logging we shouldn't do toJson
```suggestion
                                    feedRange,
```

Reply:
Done